static/src/javascripts/projects/common/modules/commercial/user-features.js

```diff
@@ -224,13 +224,27 @@ const getDaysSinceLastOneOffContribution = (): number | null => {
     return dateDiffDays(lastContributionDate, Date.now());
 };
 
-// in last six months
-const isRecentOneOffContributor = (): boolean => {
+// defaults to last six months
+const isRecentOneOffContributor = (askPauseDays: number = 180): boolean => {
     const daysSinceLastContribution = getDaysSinceLastOneOffContribution();
     if (daysSinceLastContribution === null) {
         return false;
     }
-    return daysSinceLastContribution <= 180;
+    return daysSinceLastContribution <= askPauseDays;
+};
+
+// true if the user is in the first month after ask-free period
+const isPostAskPauseOneOffContributor = (
```

Comment:
As discussed, I don't think we should use this cohort for this message test. I believe we should open the test to anyone who gave > 6 months ago to maximise the audience size (and it should be able to know if a signed in, verified user's contribution history without relying on the cookie)

Reply:
These 2 things will be addressed in follow-up PRs